bokeh/command/subcommands/serve.py

```diff
@@ -198,7 +198,7 @@ def invoke(self, args):
 
         server_kwargs = { key: getattr(args, key) for key in ['port',
                                                               'address',
-                                                              'keep_alive']
+                                                              'keep_alive_milliseconds']
                           if getattr(args, key, None) is not None }
```

Comment:
here there was another annoying problem that I wanted the flag to be `--keep-alive` then argparse sets args.keep_alive and then I want the server kwarg to have the name with _milliseconds in it.... so I have to do the rename